templates/zerver/emails/digest.source.html

```diff
@@ -17,26 +17,26 @@
                         {% endfor %}
                     </div>
                 </div>
-                {% if convo.count > 0 %}<p>+ {{ convo.count }} more message{{ convo.count|pluralize }} by {{ convo.participants|display_list(4) }}.</p>{% endif %}
+                {% if convo.count > 0 %}<p>+ {{ convo.count }} {% trans %}More message{% endtrans %}{{ convo.count|pluralize }} {% trans %}By{% endtrans %} {{ convo.participants|display_list(4) }}.</p>{% endif %}
```

Comment:
This seems wrong; we may need to read the Django docs on pluralization for how to do this correctly.

It might make sense to hold back this line since it's its own bundle of complexity for a final commit, and just try to get the rest done first.

Reply:
It was throwing errors on including the pluralization tags so I divided it. Will remove this one for now then.